cirq-core/cirq/ops/swap_gates.py

```diff
@@ -219,6 +219,11 @@ def _eigen_components(self) -> List[Tuple[float, np.ndarray]]:
         ]
         # yapf: enable
 
+    def _has_stabilizer_effect_(self) -> Optional[bool]:
+        if self._is_parameterized_():
+            return None
+        return self.exponent % 1 == 0
```

Comment:
ISwapPow gate need to have `_has_stabilizer_effect_` method just like SwapPowGate, fix it here, otherwise, unit tests would fail in pauli_string_test.py.